gcloud-java-compute/src/main/java/com/google/gcloud/compute/DeprecationStatus.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * The deprecation status associated to a Google Compute Engine resource.
+ */
+public final class DeprecationStatus<T extends ResourceId> implements Serializable {
+
+  private static final long serialVersionUID = -2695077634793679794L;
+
+  private final String deleted;
```

Comment:
Timestamps are returned as RFC strings across all the service.  If we decide for one format we need to stick to it. 

I don't have a preference, I just thought to remain close to the service representation as (milli)seconds since epoch (what we use in Storage if I recall correctly) doesn't sound much more friendly than RF  string to me. I don't mind changing so if you guys have any preference I can do it. 

Also please @aozarov let me know whenever you are done reviewing so I a can address the changes alltogether